app/src/main/java/ch/epfl/sweng/swenggolf/database/Database.java

```diff
@@ -10,6 +10,10 @@
 
 public abstract class Database {
 
+    public static final String OFFERS_PATH = "/offers";
```

Comment:
That is a very good idea to set up constants for this, but it should be changed everywhere then?

Reply:
Yes it should. I have not did it because I thought it would take me a lot of time. But it was fast using refactor options. It's now done.